src/gluonts/dataset/repository/datasets.py

```diff
@@ -152,11 +147,11 @@ def materialize_dataset(
     dataset_recipe = dataset_recipes[dataset_name]
 
     if not dataset_path.exists() or regenerate:
-        logging.info(f"downloading and processing {dataset_name}")
+        logging.info("downloading and processing %s", dataset_name)
```

Comment:
Any benefit of this over the existing usage?

Reply:
Pylint complains^^ this is the idiomatic usage for the logging package

Reply:
I still like the previous version more, since there is one less step I have to do mentally.

To hell with pylint!